slides/quickstart/index.html

```diff
@@ -6,53 +6,34 @@
   </head>
   <body>
     <p>Google Slides API Quickstart</p>
-
-    <!--Add buttons to initiate auth sequence and sign out-->
     <button id="authorize-button" style="display: none;">Authorize</button>
     <button id="signout-button" style="display: none;">Sign Out</button>
-
     <pre id="content"></pre>
 
-    <script type="text/javascript">
-      // Client ID and API key from the Developer Console
-      var CLIENT_ID = '<YOUR_CLIENT_ID>';
-      var API_KEY = '<YOUR_API_KEY>';
-
-      // Array of API discovery doc URLs for APIs used by the quickstart
-      var DISCOVERY_DOCS = ["https://slides.googleapis.com/$discovery/rest?version=v1"];
-
-      // Authorization scopes required by the API; multiple scopes can be
-      // included, separated by spaces.
-      var SCOPES = "https://www.googleapis.com/auth/presentations.readonly";
-
-      var authorizeButton = document.getElementById('authorize-button');
-      var signoutButton = document.getElementById('signout-button');
+    <script>
+      const authorizeButton = document.getElementById('authorize-button');
```

Comment:
Should these be using upper underscore casing?

https://google.github.io/styleguide/jsguide.html#formatting-braces-all

Reply:
I think `lowerCamelCase` is used in the client library and is common for DOM variables.

Reply:
Whoops, wrong link, should have been:

https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reading it more closely, these aren't constants in the traditional sense, but variables you don't intend to re-assign. Closing.